jekyll/index.html

```diff
@@ -60,5 +60,6 @@
 
   <section>
     <button data-control="gotoPage1Button" data-event-category="navigate" data-event-action="back">BACK</button>
+    <button data-control="saveAsWordDocButton" data-event-category="navigate" data-event-action="save as word doc">WORD DOCUMENT</button>
```

Comment:
I don't know if it's clear enough. I used the existing button style and that was the best thing that fit in the button. Ideally we think about how the feature works more holistically and make that change.